cypress/support/step-definitions/simple-color-picker-steps.js

```diff
@@ -47,3 +52,37 @@ When('I pick {int} color', (index) => {
     simpleColorPickerInput(i + 1).click();
   }
 });
+
+When('I pick simple {int} color', (index) => {
+  if (index === 1) {
+    experimentalSimpleColorPickerInput(SECOND_ELEMENT).click();
+    experimentalSimpleColorPickerInput(FIRST_ELEMENT).click();
+  }
+  for (let i = 0; i < index; ++i) {
+    experimentalSimpleColorPickerInput(i + 1).click();
+  }
+});
+
+Then('Experimental Simple Color Picker {int} element was picked up', (index) => {
+  experimentalSimpleColorPickerInput(index).should('have.attr', 'aria-checked', 'true');
+});
+
+When('I select {int} color', (index) => {
+  experimentalSimpleColorPickerInput(index).click();
+});
+
+When('I press downarrow on the {int} color', (index) => {
+  experimentalSimpleColorPickerInput(index).trigger('keydown', { key: 'ArrowDown', keyCode: 40, which: 40 });
+});
+
+When('I press uparrow on the {int} color', (index) => {
+  experimentalSimpleColorPickerInput(index).trigger('keydown', { key: 'ArrowUp', keyCode: 38, which: 38 });
+});
+
+When('I press rightarrow on the {int} color', (index) => {
+  experimentalSimpleColorPickerInput(index).trigger('keydown', { key: 'ArrowRight', keyCode: 39, which: 39 });
+});
+
+When('I press leftarrow on the {int} color', (index) => {
+  experimentalSimpleColorPickerInput(index).trigger('keydown', { key: 'ArrowLeft', keyCode: 37, which: 37 });
```

Comment:
the same is here, we need to use one generic step

Reply:
fixed